src/main.rs

```diff
@@ -1,9 +1,10 @@
-use std::{env,path,process};
-
+#[macro_use]
+extern crate nix;
 extern crate byteorder;
 extern crate getopts;
 extern crate libudev;
-#[macro_use] extern crate nix;
+
+use std::{env,path,process};
```

Comment:
I think the standard is putting extern first and #macro_use and similar thing at the top